.changeset/weak-grapes-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": patch
```

Comment:
```suggestion
"nextjs-website": minor
```
you are adding a new guide, so it's similar to a "new feature" then it should be a minor

Reply:
Thanks. I should have changed it, right?

Reply:
yep!